src/redux/categories/categories.js

```diff
@@ -0,0 +1,41 @@
+// categories.js
+const categoriesCollection = [];
+// Actions
+const LOAD = 'LOAD';
+const CREATE = 'CREATE';
+const CHECK_STATUS = 'CHECK_STATUS';
+const REMOVE = 'REMOVE';
```

Comment:
- [ ] The value of action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like: `const CHECK_STATUS = bookstore/categories/CHECK_STATUS`